templates/endpoints_v2/README.ipynb

```diff
@@ -20,22 +20,29 @@
     "\n",
     "## Step 1 - Run the model locally in the Workspace\n",
     "\n",
-    "The llm-serve.yaml file in this example runs the Mistral-7B model. There are 2 important configurations you would need to modify:\n",
-    "1. The `models` config in `llm-serve-*.yaml` contains a list of YAML files for the models you want to deploy. You can run any of the models in the `models` directory or define your own model YAML file and run that instead. All config files follow the naming convention `{model_name}_{accelerator_type}_{tensor_parallelism}`. Follow the CustomModels [guide](examples/CustomModels.ipynb) for bringing your own models.\n",
-    "2. `HUGGING_FACE_HUB_TOKEN` - The Meta Llama-2 family of models need the HUGGING_FACE_HUB_TOKEN variable to be set to a Hugging Face Access Token for an account with permissions to download the model.\n",
-    "\n"
+    "We provide a starter command to allow you to run Llama-2 and Mistral-family models via Ray Serve. You can specify the model ID, GPU type, tensor parallelism via the command arguments. You can also follow the [guide](examples/CustomModels.ipynb) for bringing your own models.\n",
+    "\n",
+    "Currently tensor parallelism defaults to 1 if not specified.\n",
+    "\n",
+    "Please note that the Meta Llama-2 family of models need the `hf_token` variable to be set to a Hugging Face Access Token for an account with permissions to download the model. You can get your token [here](https://huggingface.co/settings/tokens).\n"
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": null,
    "metadata": {},
+   "outputs": [],
    "source": [
-    "From the VSCode terminal (press [**Ctrl + `**] in VSCode), use the Ray Serve CLI to deploy the model for testing. It will take a few minutes to initialize and download the model.\n",
+    "# Example command to serve Mistal-7B via A10 GPUs on AWS\n",
```

Comment:
What makes this command specific to AWS? It might be confusing for a naive user because nothing in the CLI command says anything about AWS.

Reply:
A10s are available on AWS only and GCP has L4